keep/api/logging.py

```diff
@@ -126,13 +126,11 @@ def process(self, msg, kwargs):
         extra["tenant_id"] = self.tenant_id
         extra["workflow_id"] = self.workflow_id
         extra["workflow_execution_id"] = self.workflow_execution_id
-        # add the steps/actions context
-        # todo: more robust
-        # added: protection from big steps context (< 64kb)
-        if self.context_manager.steps_context_size < 1024 * 64:
-            extra["context"] = (self.context_manager.steps_context,)
-        else:
-            extra["context"] = "truncated (context size > 64kb)"
+        # For now, just adding the step_id to the context, so fronted can show step's results
+        extra["context"] = {}
+        step_id = extra.pop("step_id", None)
+        if step_id:
+            extra["context"]["step_id"] = step_id
```

Comment:
why its in context?

why not just `extra["step_id"] = step_id`?

Reply:
it's because in DB we have `context` column, and I want to save step_id to this column. however, maybe it's better to just save remaining keys of extra to `context`